Mihai-Blaga/COMP30022
# Tasks/AA-87 delete contact

## Changes
- Added delete contact button in fragment xml
- Created contact function in Contact
## Tests
- Click on any contact in contacts list
- Press delete button (The app may freeze for a bit)
- Verify that event has been deleted in CRM app and Google Contacts
